### ISSUE TITLE
Subnet-beta module does not support REGIONAL_MANAGED_PROXY

### ISSUE THREAD

**Issue by CoreDaveP** [terraform-google-modules/terraform-google-network#371]
### TL;DR

As per docs here:

https://cloud.google.com/load-balancing/docs/proxy-only-subnets

Attempting to do this with the subnets-beta module results in an error...

Error: Error creating Subnetwork: googleapi: Error 400: Invalid value for field 'resource.logConfig': '{  "enable": false}'. Invalid field set in subnetwork with purpose REGIONAL_MANAGED_PROXY., invalid

### Expected behavior

Successful creation of REGIONAL_MANAGED_PROXY subnet.

### Observed behavior

Error: Error creating Subnetwork: googleapi: Error 400: Invalid value for field 'resource.logConfig': '{  "enable": false}'. Invalid field set in subnetwork with purpose REGIONAL_MANAGED_PROXY., invalid

### Terraform Configuration

```hcl
Subnet Config as follows :
        {  
          "subnet_name": "snet-01",
          "subnet_ip": "10.10.10.0/24",
          "subnet_region": "europe-west2",
          "description" : "Envoy Proxy Subnet",
          "purpose" :  "REGIONAL_MANAGED_PROXY",
          "role" : "ACTIVE"       
        },
```

### Terraform Version

```sh
$ terraform --version
Terraform v1.0.5
```

**Comment by bharathkkb** [comment]
@CoreDaveP It looks like a provider issue as from your config the `logConfig` block should be empty.

**Comment by CoreDaveP** [comment]
@bharathkkb - Apologies, you are correct...updated the provider and the module worked fine - thank you!
